python/lsst/ts/aos/utils/m1m3/m1m3_aav.py

```diff
@@ -361,6 +370,61 @@ def vect_to_fam(row: pd.Series) -> pd.Series:
         )
         self.calculated_fam.set_index(self.fitter.aav.index, inplace=True)
 
+    async def load_accelerometers(self, start: Time, end: Time) -> None | pd.DataFrame:
+        """Load DC accelerometers data in given interval. Those are used for
+        acceleration  corrections.
+
+        Parameters
+        ----------
+        start : Time
+            Search interval start time.
+        end : Time
+            Search interval end time.
+
+        Returns
+        -------
+        ret : pd.DataFrame
```

Comment:
since ret doesn't reall mean anything, maybe you can just remove "ret :" and leave the object type

Reply:
Changed to accelerometers. This is numpy doc, you have to provide return names.